website/docs/d/schema_site_vrf_region.html.markdown

```diff
@@ -27,26 +27,26 @@ data "mso_schema_site_vrf_region" "vrfRegion" {
 
 * `schema_id` - (Required) SchemaID under which you want to deploy Vrf Region.
 * `site_id` - (Required) SiteID under which you want to deploy Vrf Region.
+* `template_name` - (Required) Template where Vrf Region to be created.
```

Comment:
Template where Vrf Region to be created. -> Template under which you want to create/deploy Vrf Region.

Just to keep it consistent to the other attribute description

Reply:
ok